core/pom.xml

```diff
@@ -100,45 +130,39 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>software.amazon.awssdk</groupId>
-            <artifactId>http-client-spi</artifactId>
-            <version>${awsjavasdk.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>software.amazon.awssdk</groupId>
-            <artifactId>apache-client</artifactId>
-            <version>${awsjavasdk.version}</version>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
-            <artifactId>utils</artifactId>
+            <artifactId>test-utils</artifactId>
             <version>${awsjavasdk.version}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.reactivestreams</groupId>
-            <artifactId>reactive-streams</artifactId>
+            <artifactId>reactive-streams-tck</artifactId>
+            <scope>test</scope>
         </dependency>
-
         <dependency>
-            <groupId>org.assertj</groupId>
-            <artifactId>assertj-core</artifactId>
+            <groupId>com.google.jimfs</groupId>
```

Comment:
What's this?

Reply:
This is used in TCK tests, https://github.com/aws/aws-sdk-java-v2/blob/master/core/src/test/java/software/amazon/awssdk/core/async/FileAsyncRequestPublisherTckTest.java